srcpkgs/ergo/files/ergo/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+[ -r ./conf ] && . ./conf
+
+exec 2>&1
+exec chpst -u _ergo:_ergo ergo run --conf /etc/ergo.conf
```

Comment:
If this is going to source a configuration, that configuration might as well provide the opportunity to override the configuration file:
```suggestion
exec chpst -u _ergo:_ergo ergo run ${OPTS:---conf /etc/ergo.conf}
```